pkg/speaker/bgp/conf.go

```diff
@@ -19,7 +20,15 @@ func (b *Bgp) HandleBgpGlobalConfig(global *bgpapi.BgpConf, rack string, delete
 	}
 
 	b.bgpServer.StopBgp(context.Background(), nil)
-	return b.bgpServer.StartBgp(context.Background(), &api.StartBgpRequest{
+	err = b.bgpServer.StartBgp(context.Background(), &api.StartBgpRequest{
 		Global: request,
 	})
+	if err != nil {
+		return err
+	}
+
+	if cm != nil {
+		b.UpdatePolicy(cm)
```

Comment:
Why ignore the return value of this function?

Reply:
We need to discuss whether error should be ignored here?

/cc @renyunkang @amal-thundiyil

Reply:
We can detect whether `cm` is `nil` in `updatePolicy`. Returns `updatePolicy` directly here.

Reply:
I mean, if the user configures the `policy` incorrectly. Do we need to re-queue the request?

Reply:
I think print out the error log, no need for re-queue.